### PR TITLE
[networking_mapping] Add DNS field to the networking definition.

### PR DESCRIPTION
**Pull request by pablintino**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

**Comment by pablintino** [comment]
recheck

**Comment by son-vyas** [comment]
@pablintino can you please rebase this pr ?

File: ci_framework/plugins/module_utils/net_map/networking_definition.py [hunk 1]
```diff
@@ -887,6 +1020,7 @@ class NetworkDefinition:
     __FIELD_NETWORK_IPV4 = "network-v4"
     __FIELD_NETWORK_IPV6 = "network-v6"
     __FIELD_GATEWAY = "gateway"
+    __FIELD_DNSS = "dns"
```
Comment (rlandy) [inline ci_framework/plugins/module_utils/net_map/networking_definition.py:1023]: DNSS - intentional?

**Comment by pablintino** [comment]
recheck

**Pull request merged**